EIPS/eip-233.md

```diff
@@ -54,7 +54,7 @@ The Meta EIP representing the hard fork should move in to the `Accepted` state o
 
 A template for the [Istanbul Hardfork Meta 1679](https://eips.ethereum.org/EIPS/eip-1679) is included below ([source file on Github](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1679.md)):
 
-```
+```markdown
 {% raw %}
```

Comment:
Not sure about this, given it has this "raw" modifier?

Reply:
Yes. Linguist has improved to support Front Matters in ` ` ` markdown.

But thanks for pointing out -- cleaned up at dce9af347852e434c19d75607f27d75fa850401e

Reply:
Can we handle this file in a separate PR? I'm not sure what is the good solution for it, but don't want to delay this PR.